app/src/app/components/contents/packets/PacketDependencies.tsx

```diff
@@ -18,10 +20,20 @@ export const PacketDependencies = ({ depends }: PacketDependenciesProps) => {
         </AccordionTrigger>
         <AccordionContent>
           <ul className="space-y-1 overflow-y-auto max-h-80">
-            {depends.length === 0 ? (
-              <div className="italic text-sm">This packet has no dependencies on other packets</div>
+            {dependencies && dependencies.length === 0 ? (
+              <div className="italic text-sm">None</div>
```

Comment:
Updated text to match that for the other sections on this page (parameters, and something else I forgot) which show 'None' when empty.